playbooks/roles/logger/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+logger_address: 192.168.9.90
```

Comment:
Let's also use capital letters for these vars, as they're intended to be overridden.